Description: Remove bundler requirement
 Remove the bundler dependency from test file.
 .
 ruby-redis-activesupport (4.0.0-1) UNRELEASED; urgency=medium
 .
   * Initial release (Closes: #nnnn)
Author: Balasankar C <balasankarc@autistici.org>


--- ruby-redis-activesupport-4.0.0.orig/test/test_helper.rb
+++ ruby-redis-activesupport-4.0.0/test/test_helper.rb
@@ -1,4 +1,4 @@
-require 'bundler/setup'
+#require 'bundler/setup'
 require 'minitest/autorun'
 require 'mocha/setup'
 require 'active_support'
